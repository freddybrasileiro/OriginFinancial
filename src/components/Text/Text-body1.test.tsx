import * as React from 'react';
import { shallow } from 'enzyme';
import { Text } from './index';

describe('Text', () => {
  it('Should render the Text component with variant body1', () => {
    expect(shallow(<Text variant="body1">My label</Text>)).toMatchSnapshot();
  });
});
