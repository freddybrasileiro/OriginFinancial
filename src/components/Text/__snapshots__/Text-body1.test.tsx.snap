// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text Should render the Text component with variant body1 1`] = `ShallowWrapper {}`;
